vnext/ReactCommon/ReactCommon.vcxproj

```diff
@@ -189,6 +213,23 @@
     <CLCompile Include="$(ReactNativeDir)\ReactCommon\react\nativemodule\samples\ReactCommon\NativeSampleTurboCxxModuleSpecJSI.cpp" />
     <CLCompile Include="$(ReactNativeDir)\ReactCommon\react\nativemodule\samples\ReactCommon\SampleTurboCxxModule.cpp" />
   </ItemGroup>
+  <ItemGroup>
+    <CLCompile Include="$(ReactNativeDir)\ReactCommon\hermes\inspector\Inspector.cpp" />
```

Comment:
>Inspector.cpp" /> [](start = 71, length = 17)

Ideally we must put all the Hermes related code into its own DLL.
It looks a little bit odd that even if we use Chakra we still compile in all this Hermes related code.

Reply:
We are doing that now !